src/wp-admin/includes/class-wp-media-list-table.php

```diff
@@ -1233,11 +1233,11 @@ public function inline_edit() {
 									<label for="quick-day">
 										<span class="screen-reader-text"><?php echo esc_html_e( 'Day' ); ?></span>
 									</label>
-									<input id="quick-day" type="text" name="jj" value="" size="2" maxlength="2" autocomplete="off" class="form-required">&nbsp;,
+									<input id="quick-day" type="text" name="jj" value="" size="2" maxlength="2" inputmode="numeric" pattern="(0?[1-9]|[12][0-9]|3[01])$"autocomplete="off" class="form-required">&nbsp;,
```

Comment:
Seems to be missing a space between the `pattern` parameter ending and the start of `autocomplete`.

Reply:
Does the pattern need to start with `^`?

Reply:
That's what I thought. Don't we want full-string matching? And if we don't, why not?